Kernel/Arch/init.cpp

```diff
@@ -30,7 +30,7 @@
 #include <Kernel/Devices/KCOVDevice.h>
 #include <Kernel/Devices/PCISerialDevice.h>
 #include <Kernel/Devices/SerialDevice.h>
-#include <Kernel/Devices/Storage/StorageManagement.h>
+#include <Kernel/Devices/Storage/Management.h>
```

Comment:
Personal taste: I like having class names reflected in the filenames whenever possible. `Storage/Management.h` seems like such a generic name that I'd come to expect a namespaced `Storage::Management` or similar, but the class is still called `StorageManagement`. I _think_ we try to keep the names the same as much as possible, and I'm not sure if it's worth diverging from that pattern here.

Reply:
From what I saw, we actually remove redundant words. See the Audio subsystem for example.
I rather have it this way (removing redundant words) to be honest, seems more clean in my opinion.

Reply:
Re-thinking about this, the namespace reason is pretty strong... but we should decide on one scheme and propagate it to all subsystems.

Reply:
Hm, I could find some other classes that use a shorter filename than the actual class name, but those were all written by you. Do you have some examples of diverging from the class name that wasn't written by you?

Reply:
At any rate, I think using namespaces for this makes the most sense and looks the best. There's is no need to actively propagate it to all systems, we can just decide on a style, document that style and make sure to only accept PRs implementing that new style. </trying_to_prevent_100s_of_refactor_prs>